clients/web/static/lib/bootstrap/fonts/glyphicons-halflings-regular.svg

```diff
@@ -0,0 +1,229 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
Experiment complete, we just need to pass the following value into the widget's options when creating it:


    icons: {
                time: 'icon-clock',
                date: 'icon-calendar',
                up: 'icon-up-open',
                down: 'icon-down-open',
                previous: 'icon-left-open',
                next: 'icon-right-open',
                clear: 'icon-trash',
                close: 'icon-cancel',
                today: 'icon-target'
            }

I'm really impressed with this library, it's got great customizability, has a really nice look and feel and interaction, MIT license, it's in npm and is well maintained... all in all, you made a great choice amongst the hundreds of libraries that do the same thing. Nice work.